content/en/api/notifications.md

```diff
@@ -1,8 +1,210 @@
 ---
-title: Notification Services
+title: Notifications / Webhooks
 description: ""
 position: 42
 category: API
 ---
 
-Content
\ No newline at end of file
+## What are notifications and why they are useful
+Easypay use notifications (webhooks) for event notifications.
+
+Notifications are **POST API** calls (JSON encoded) that let your application know an event has happened.
+
+This is a server-to-server service that sends near real-time notifications, informing about a payment status change.
+
+Notifications are particularly useful for asynchronous events like:
+
+* when a customer pays a MB reference
+* a frequent payment succeeds/fails.
+* subscription succeeds/fails, among other useful events.
+* a checkout single sale payment succeeds/fails.
+
+With notifications your system can automate custom actions in response to the transitions that happen in a payment flow.
+
+
+## Supported Notification types
+
+Easypay supports 3 notification types:
+
+- **Generic notification**: a notification informing about all state transitions of some payment resource.
+- **Authorisation notification**: a notification informing about some payment authorization status. Available for single and frequent payments.
```

Comment:
So when the state changes to authorisation we receive two different notifications? I think this could be clarified.

Reply:
if we subscribed to both, yes, we will receive two notifications.

Reply:
please re-check... i tried to address your points.